### PR TITLE
fixes authentication and removes console logs

### PR DESCRIPTION
**Pull request by NNKamel**

### CODE REVIEW

**Review by clarson217 — APPROVED** [review]

File: src/app/app.component.html [hunk 1]
```diff
@@ -9,43 +9,132 @@
...
+            <!-- svg source https://freesvg.org/vector-clip-art-of-minimal-design-search-icon -->
+            <svg xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cc="http://creativecommons.org/ns#"
+              xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:svg="http://www.w3.org/2000/svg"
+              xmlns="http://www.w3.org/2000/svg" xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+              xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape" width="25" height="20"
+              viewBox="0 0 55.770032 124.72441" id="svg4983" version="1.1" inkscape:version="0.91 r13725"
+              sodipodi:docname="Minimal Magnifying Glass.svg">
+              <g inkscape:label="Layer 1" inkscape:groupmode="layer" id="layer1"
+                transform="translate(64.082039,-216.03793)">
+                <g id="g4288" transform="matrix(12.843054,0,0,12.843054,-4352.973,-5791.3727)" style="
```
Comment (markmoorerev) [inline src/app/app.component.html]: should all this be in a .css file?

File: src/app/logger.service.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoggerService {
```
Comment (markmoorerev) [inline src/app/logger.service.ts:6]: is this going to be modified to do real logging eventually?

File: src/app/movie/movie.component.ts [hunk 3]
```diff
@@ -35,106 +36,100 @@ export class MovieComponent implements OnInit {
   reviewsBusy: boolean = false;
   lastPage: boolean = false;
 
-  sumbitReview: any ={
-    rating:0,
-    movieid:this.router.snapshot.params.id,
-    username:0,
-    text:""
+  sumbitReview: any = {
```
Comment (markmoorerev) [inline src/app/movie/movie.component.ts]: is there no defined model for these?

**Review by markmoorerev — CHANGES_REQUESTED** [review]
Every place where you are using any as the type, create a defined model (if possible).

File: src/app/app.component.html (hunk 1 above)
Comment (NNKamel) [reply src/app/app.component.html]: @tristyn-l resolved this

File: src/app/logger.service.ts (hunk 2 above)
Comment (NNKamel) [reply src/app/logger.service.ts:6]: Yes this should change.

**Review by markmoorerev — APPROVED** [review]

**Pull request merged**